packages/functional-tests/lib/targets/stage.ts

```diff
@@ -5,7 +5,7 @@ export class StageTarget extends RemoteTarget {
   static readonly target = 'stage';
   readonly name: TargetName = StageTarget.target;
   readonly contentServerUrl = 'https://accounts.stage.mozaws.net';
-  readonly relierUrl = 'https://stage-123done.herokuapp.com';
+  readonly relierUrl = 'https://stage-123done.herokuapp.com/';
```

Comment:
Q: Why did this need a trailing slash (but the `contentServerUrl` doesn't)?

Reply:
Yeah, good catch! I'll remove it.